themes/theme-rui/src/components/List.styles.ts

```diff
@@ -1,7 +1,30 @@
 import { ThemeComponent, cva } from '@marigold/system';
 
 export const List: ThemeComponent<'List'> = {
-  ul: cva('ml-6 list-outside list-disc'),
-  ol: cva('ml-6 list-outside list-decimal'),
-  item: cva('not-first:pt-3'),
+  ul: cva('ml-6 list-outside list-disc space-y-1 marker:text-foreground/50', {
```

Comment:
why `foreground/50`? Thought we do no percent values from colors, should be `muted`?

Reply:
Was too light 🫣

Reply:
do we need a new token then